### ISSUE TITLE
Ark 3.06.15 - Default Controls - Guild Bank Only

### ISSUE THREAD

**Issue by Sekhy** [arkayenro/arkinventory#786]
Odd one to explain this.  Ark was working fine this morning.  Logged on this evening and under the Controls screen under Blueprint Style and Layer the only options I have for every tab are;
[1001] Default
[9998] * Guild Bank
[9999] * Default

I no longer have the option to select Bag, Guild Bank or Mount etc. like I did before.  Now when I try to configure my bags it changes my guild bank!  I've reinstalled Ark Inventory and it installs the same way.  My friend's account is fine and working perfectly.

Any ideas?

Thanks

**Comment by arkayenro** [comment]
can you restore a back from 30512 and run it through again?  try .15 and see if that makes any difference

**Comment by Yanngo** [comment]
You need different layouts for your bag and your bank. The **designs** tab should be named **Layouts** I strongly suspect.
![style_wowscrnshot_072116_092724](https://cloud.githubusercontent.com/assets/17704069/17062489/cf482e0a-503b-11e6-89f2-951bdb411257.jpg)

And in the bag tab and the bank tab you need different layouts in the dropdown menu
![wowscrnshot_072016_110317](https://cloud.githubusercontent.com/assets/17704069/17062539/f8b0a952-503b-11e6-83f4-a7c1ad2e3385.jpg)

**Comment by arkayenro** [comment]
the design tab will be renamed to "styles / layouts" so its a bit more obvious

**Comment by Sekhy** [comment]
Hi Vango, Arkayenro
Yes it's this bit I'm having trouble with.  I don't have the option to select different Styles for different tabs. I have [1000] Default, [9998] Guild Bank and [9999] Default in the type drop down on ALL the tabs.  I used to have all the different options and everything worked fine.  Now the only thing I can select for any tab is those three options.  
I will try the back up.
Thanks
Sekhy

**Comment by arkayenro** [comment]
can you see them listed in the category set and style/layout tabs?

**Comment by Sekhy** [comment]
Hi Arkayenro

As it happens, the latest update appears to have fixed the issue.  As I say, I happened randomly between logins so I have kept the back-up as a stand by.  Even a reinstall of the addon didn't fix it.

To answer your question though, no I couldn't seen anything in the style/layout tabs but for those three options.  Now, however, I can see all of them again!

Thanks for your help and a great addon :)

Sekhy
